static/sass/custom/_homepage-hero-promo.scss

```diff
@@ -19,43 +19,20 @@
 }
 
 .homepage-hero-image-wrap {
-  min-height: 16rem;
-  position: relative;
+  position: static;
 
-  @media (min-width: $breakpoint-medium) {
-    position: static;
+  @media (max-width: $breakpoint-medium) {
+    display: none;
```

Comment:
We can use the `u-hide--medium u-hide--small` classes to achieve this.
https://docs.vanillaframework.io/en/utilities/hide

Reply:
Thanks, good to know! This is no longer being hidden, but I'll remember for next time.